nschecker/notify.go

```diff
@@ -58,8 +57,24 @@ func (n *SlackNotifier) Notify(state State, s Source) error {
 		channel = "<!channel|channel> "
 	}
 	msg := fmt.Sprintf("%s%v: %v (%v)", channel, state, s.URL, s.Name)
-	params := slack.PostMessageParameters{EscapeText: false}
-	_, _, err := n.Cli.PostMessage(n.channel, msg, params)
+	v := url.Values{}
```

Comment:
why did you change SlackNotifier?

Please add comment or describe the reason in commit message.